osbuild/util/ostree.py

```diff
@@ -17,7 +18,7 @@ def check(self, value):
         origin = getattr(self.type, "__origin__", None)
         if origin:
             self.typecheck(value, origin)
-            if origin is list:
+            if origin is list or typing.List:
```

Comment:
Shouldn't this be `origin is list or origin is typing.List`? Also, shouldn't we use `isinstance()`?

Reply:
Doh. Yes. It was late.

Reply:
Ok, nothing to see here, move along ...
FTR: I wanted exact matches, therefore no `isinstance()`.